tests/python/mock.py

```diff
@@ -1,22 +1,40 @@
 # mock.py
 # Test tools for mocking and patching.
-# Copyright (C) 2007-2012 Michael Foord & the mock team
 # E-mail: fuzzyman AT voidspace DOT org DOT uk
-
-# mock 0.8.0
+#
```

Comment:
Integrated the latest version of mock.py that is compatible with Python 2.5. Also, this one has support for PropertyMock, which is useful for mocking properties on Python objects, which I'm actually using in test_user_settings.py